pkg/test/expectations/expectations.go

```diff
@@ -208,8 +208,12 @@ func ExpectMetric(prefix string) *prometheus.MetricFamily {
 }
 func ExpectManualBinding(ctx context.Context, c client.Client, pod *v1.Pod, node *v1.Node) {
 	Expect(c.Create(ctx, &v1.Binding{
-		TypeMeta:   pod.TypeMeta,
-		ObjectMeta: pod.ObjectMeta,
+		TypeMeta: pod.TypeMeta,
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
Curious what you need to strip from the objectmeta

Reply:
The 1.23 API server fails if there is an existing resource version on an object's metadata during create.